src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double DEFAULT_VALUE = 10.0;
+    private static final int MULTIPLAYER = 100;
+    private ColorSupplier colorSupplier;
+    private Random random = new Random();
+
+    public FigureSupplier() {
+        this.colorSupplier = new ColorSupplier();
+    }
+
+    public Figure getRandomFigure() {
+        FiguresShape[] figureshape = FiguresShape.values();
+        Figure fg = null;
+        switch (figureshape[random.nextInt(figureshape.length)].name()) {
+            case "SQUARE": {
```

Comment:
Use enums instead of String representation.

Reply:
not fixed